pkg/apis/core/v1alpha1/federatedtypeconfig_types.go

```diff
@@ -78,7 +79,18 @@ type FederatedTypeConfigStatus struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// FederatedTypeConfig
+// FederatedTypeConfig programs federation to know about a single API resource -
+// the "target resource" - that a user wants to federate. For each target resource,
+// there is a set of API resources that capture the information required to
```

Comment:
s/is/are/

Reply:
'is' is correct because 'it' refers to the set, but I won't die on this hill :)